packages/genesys-spark-components/src/components/stable/gux-listbox-multi/gux-option-multi/gux-option-multi.tsx

```diff
@@ -118,6 +124,35 @@ export class GuxOptionMulti {
     }
   }
 
+  private renderText(): JSX.Element {
+    // The gux-slot-container attribute is used in gux-listbox-multi and gux-dropdown-multi as a selector to get the slotted gux-option-multi text.
+    // This attribute is required because we need to get the slotted text and exclude the screen reader text.
+    if (this.hasSubtext) {
+      return (
+        <div class="gux-option-text">
```

Comment:
is this supposed to be `gux-option-wrapper`? I don't see gux-option-text in the scss file

Reply:
Good spot